### ISSUE TITLE
numerically approximate conformal maps from arbitrary domains to the unit disc using the circle packing method

### ISSUE THREAD

**Issue by adammaj1** [johncbowers/koebepy#1]
Is it possible to 
[numerically approximate conformal maps from arbitrary domains to the unit disc](https://math.stackexchange.com/questions/1885066/is-it-possible-to-find-aproximation-of-conformal-map-from-sequences-of-complex-p) using koebepy?

**Comment by johncbowers** [comment]
It should be, let me see if I can write an example script.

**Comment by johncbowers** [comment]
How do you want to input your domain? Is a polygon sufficient?

**Comment by adammaj1** [comment]
Yes. See for example : https://math.stackexchange.com/questions/1885066/is-it-possible-to-find-aproximation-of-conformal-map-from-sequences-of-complex-p
